conf/layer.conf

```diff
@@ -15,13 +15,12 @@ BBFILE_PATTERN_pelux-layer := "^${LAYERDIR}/"
 
 BBFILE_PRIORITY_pelux-layer = "8"
 
-# Let us add layer-specific bbappends which are only applied when that
-# layer is included in our configuration
-BBFILES += "${@' '.join('${LAYERDIR}/layers/%s/recipes*/*/*.bbappend' % layer \
-               for layer in BBFILE_COLLECTIONS.split())}"
-# Do the same for recipes
-BBFILES += "${@' '.join('${LAYERDIR}/layers/%s/recipes*/*/*.bb' % layer \
-               for layer in BBFILE_COLLECTIONS.split())}"
+BBFILES_DYNAMIC += " \
+    b2qt:${LAYERDIR}/dynamic-layers/b2qt-layer/*/*/*.bb \
```

Comment:
consider rename `b2qt-layer` to `meta-boot2qt` to be consistent with the naming of the parent layer.

Reply:
this seems to be an established syntax:
- https://github.com/agherzan/meta-raspberrypi/tree/master/dynamic-layers
- https://github.com/Freescale/meta-freescale/tree/master/dynamic-layers

Reply:
I had a look into the reference manual: https://yoctoproject.org/docs/latest/ref-manual/ref-manual.html#var-BBFILES_DYNAMIC
and you can see that the parent layer for `clang-layer` is `meta-clang`, and for `core` it's `openembedded-core`, and not `clang-layer` and `core-layer`, that's why I commented this.

But I'm fine with `xx-layer` instead of `meta-xx`.

However, I think `boot2qt-layer` is more appropriate than `b2qt-layer? , don't you think?

Reply:
I believe it comes from `BBFILE_COLLECTIONS` of the layer.